### PR TITLE
Update design of Manage Users page

### PR DESCRIPTION
**Pull request by janfaracik**
Small PR which removes the side panel from the users page, moves the 'Create user' button to the app bar and removes the unnecessary styling on the 'Create user' form.

**Before**
image

image

**After**
image

image

### Proposed changelog entries

- Update design of Manage Users page

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [ ] (If applicable) Jira issue is well described
- [ ] Changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developer, depending on the change) and are in the imperative mood. [Examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)
  - Fill-in the `Proposed changelog entries` section only if there are breaking changes or other changes which may require extra steps from users during the upgrade
- [ ] Appropriate autotests or explanation to why this change has no tests
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadoc, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")` if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content-Security-Policy directives (see [documentation on jenkins.io](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates: links to external changelogs and, if possible, full diffs

### Desired reviewers

@jenkinsci/sig-ux 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least 2 approvals for the pull request and no outstanding requests for change
- [ ] Conversations in the pull request are over OR it is explicit that a reviewer does not block the change
- [ ] Changelog entries in the PR title and/or `Proposed changelog entries` are accurate, human-readable, and in the imperative mood
- [ ] Proper changelog labels are set so that the changelog can be generated automatically
- [ ] If the change needs additional upgrade steps from users, `upgrade-guide-needed` label is set and there is a `Proposed upgrade guidelines` section in the PR title. ([example](https://github.com/jenkinsci/jenkins/pull/4387))
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7197"></a>

### CODE REVIEW

File: core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/index.jelly
Comment (timja) [inline core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/index.jelly:27]: do you think the form could do with a max-width? it seems very wide now

image

**Review by timja — APPROVED** [review]
nice work

**Review by NotMyFault — APPROVED** [review]

File: core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/index.jelly
Comment (uhafner) [reply core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/index.jelly:27]: Also does this button use primary button class? I don't see this form in the PR?

Comment (janfaracik) [reply core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/index.jelly:27]: > Also does this button use primary button class? I don't see this form in the PR?

The submit button used the `<f:submit />` component which currently uses the YUI classes, I've got a branch in the works to update that component 👍

Comment (janfaracik) [reply core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/index.jelly:27]: > do you think the form could do with a max-width? it seems very wide now
> 
> image

Updated - 

image

File: core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/_entryFormPage.jelly
Comment (timja) [inline core/src/main/resources/hudson/security/HudsonPrivateSecurityRealm/_entryFormPage.jelly:50]: Looks a bit off for me on both big screens and mobile

27" monitor
image

iphone XR from chrome devtools:
image

would centring it help maybe?

**Review by timja — APPROVED** [review]
the margin auto changes look a lot better, thanks!

**Review by yaroslavafenkin — COMMENTED** [review]
I have mixed feelings about the change.
"Create user" page looks better than before, but max width limitation feels awkward to me and it's inconsistent with other pages.
Not a big fan of removing "Create user" item from the side panel. To me it's intuitive to seek for actions relevant to a certain page on the left.

Have no security concerns, so I'll adjust the labels.

File: war/src/main/less/abstracts/theme.less
Comment (daniel-beck) [inline war/src/main/less/abstracts/theme.less:271]: This change and the following seem independent of the Manage Users page?

**Comment by timja** [comment]
> This change and the following seem independent of the change to the Manage Users page and should be a separate PR.

The two pages are related, sure it could have been separate but it could also have been combined

----

/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Pull request merged**
